coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/universe/sql/SqlTableCleanup.scala

```diff
@@ -2,10 +2,11 @@ package com.socrata.datacoordinator.truth.universe
 package sql
 
 import java.sql.Connection
+import scala.concurrent.duration.FiniteDuration
 
 import com.rojoma.simplearm.util._
 
-class SqlTableCleanup(conn: Connection, daysDelay: Int = 1) extends TableCleanup {
+class SqlTableCleanup(conn: Connection, daysDelay: FiniteDuration) extends TableCleanup {
   val log = org.slf4j.LoggerFactory.getLogger(classOf[SqlTableCleanup])
   def cleanupPendingDrops(): Boolean = {
     using(conn.createStatement()) { stmt =>
```

Comment:
Does the SELECT query actually end up being valid now that you've changed the type from `Int` to `FiniteDuration`? A good way to test this would be to change your config value to something small like 10 seconds, create and delete a dataset, and see whether data coordinator actually drops the table without error.